WalletWasabi.Tests/HwiTests/NoDeviceConnectedTests/HwiProcessBridgeMock.cs

```diff
@@ -0,0 +1,310 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Hwi2.Models;
+using WalletWasabi.Hwi2.Parsers;
+using WalletWasabi.Interfaces;
+
+namespace WalletWasabi.Tests.HwiTests.NoDeviceConnectedTests
+{
+	public class HwiProcessBridgeMock : IProcessBridge
+	{
+		public HardwareWalletModels Model { get; }
+
+		public HwiProcessBridgeMock(HardwareWalletModels model)
+		{
+			Model = model;
+		}
+
+		public Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel)
+		{
+			if (openConsole)
+			{
+				throw new NotImplementedException($"Cannot mock {nameof(openConsole)} mode.");
+			}
+
+			string type = null;
+			string rawPath = null;
+
+			if (Model == HardwareWalletModels.TrezorT)
+			{
+				type = "trezor";
+				rawPath = "webusb: 001:4";
+			}
+			else if (Model == HardwareWalletModels.TrezorOne)
+			{
+				type = "trezor";
+				rawPath = "hid:\\\\\\\\?\\\\hid#vid_534c&pid_0001&mi_00#7&6f0b727&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}";
+			}
+			else if (Model == HardwareWalletModels.ColdcardMk1)
+			{
+				type = "coldcard";
+				rawPath = @"\\\\?\\hid#vid_d13e&pid_cc10&mi_00#7&1b239988&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}";
+			}
+			else if (Model == HardwareWalletModels.LedgerNanoS)
+			{
+				type = "ledger";
+				rawPath = "\\\\\\\\?\\\\hid#vid_2c97&pid_0001&mi_00#7&e45ae20&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}";
+			}
+			else
+			{
+				throw new NotImplementedException("Mock missing");
+			}
+
+			string path = HwiParser.NormalizeRawDevicePath(rawPath);
+			string devicePathAndTypeArgumentString = $"--device-path \"{path}\" --device-type \"{type}\"";
+
+			const string successTrueResponse = "{\"success\": true}\r\n";
+
+			if (CompareArguments(arguments, "enumerate"))
+			{
+				if (Model == HardwareWalletModels.TrezorT)
+				{
+					var response = $"[{{\"type\": \"{type}\", \"path\": \"{rawPath}\", \"needs_pin_sent\": false, \"needs_passphrase_sent\": false, \"error\": \"Not initialized\"}}]";
+					var code = 0;
```

Comment:
This variable can be declared once and used when needed.

Reply:
Good point but that's a lot. I will optimize the code later at the end.

Reply:
Done

Reply:
@nopara73 Great job!